drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -324,6 +363,15 @@ static void setup_mmal_port_format(struct bcm2835_isp_node *node,
 	port->es.video.crop.height = q_data->height;
 	port->es.video.crop.x = 0;
 	port->es.video.crop.y = 0;
+
+	mmal_colour_space = translate_color_space(q_data->colorspace);
```

Comment:
Quicker and easier to just return ```MMAL_COLOR_SPACE_UNKNOWN``` from ```translate_color_space```, and then that can be assigned direct to ```port->es.video.color_space```. Does the error message really add anything? Shouldn't try_fmt have filtered that out already? Or put the error message in ```translate_color_space```